app/services/forest_liana/resources_getter.rb

```diff
@@ -19,6 +19,19 @@ def perform
         @records = @records.where(@segment.where.call())
       end
 
+      # Live Query mode
+      if @params[:query]
```

Comment:
very hard to understand that it is in a context of a Live Query Segment.

Reply:
There's a comment for this.